tests/installation/await_install.pm

```diff
@@ -104,6 +104,10 @@ sub run {
             assert_screen \@tags, $timeout;
         }
 
+        if (match_has_tag('debugger_popup_warning')) {
```

Comment:
Let's name needle `yast-error`. We have same approach for autoyast, there needle is called `autoyast-error` which we can rename then.